common/domain/handler_test.go

```diff
@@ -2433,7 +2433,7 @@ func TestHandler_UpdateDomain(t *testing.T) {
 				Name:                     constants.TestDomainName,
 				FailoverTimeoutInSeconds: common.Int32Ptr(1),
 			},
-			err: errInvalidGracefulFailover,
+			err: errInvalidFailoverNoChangeDetected,
```

Comment:
this is not only a graceful error condition, it refers to any change which fails validation or changes, so updating here